region/lowernorfair/east.json

```diff
@@ -7869,9 +7951,68 @@
               "id": 2,
               "strats": [
                 {
-                  "name": "Base",
+                  "name": "Itemless Damageless",
                   "notable": false,
-                  "requires": []
+                  "requires": [
+                    "canInsaneWalljump",
```

Comment:
Should this be in an `or`? This strat will be turned off if wall jumps are disabled. Maybe we should eventually have a way to indicate if a strat is notable, only if wall jumps are disabled.

Reply:
The difficulty of the strat is at insaneWalljump level but you can do it with either a jump or walljump.
Suppose I'll make it notable.